designs/multi-cluster.md

```diff
@@ -35,6 +37,10 @@ Multi-cluster use-cases require the creation of multiple managers and/or cluster
 objects. This proposal is about adding native support for multi-cluster use-cases
 to controller-runtime.
 
+With this change, it will be possible to implement pluggable cluster providers
+that automatically start and stop watches (and thus, cluster-aware reconcilers) when
```

Comment:
watches or sources?

Reply:
Good point, it's `ctxBoundedSyncingSources` that are being started ...